allensdk/brain_observatory/nwb/__init__.py

```diff
@@ -27,6 +27,25 @@
 log = logging.getLogger("allensdk.brain_observatory.nwb")
 
 
+cell_specimen_col_descriptions = {
```

Comment:
Since this looks to be a module level constant it should be in ALL CAPS

Reply:
changed to all caps for module constant